src/components/FormFields/ParameterFields.js

```diff
@@ -22,20 +39,27 @@ const ParameterFields = ({ jobType }) => {
     }
 
     // Map all parameters to the appropriate field types
-    const parameterComponents = parameters.map(({ fieldName, name, klass }) => {
+    const parameterComponents = parameters.map(({ fieldName, name, klass, ...rest }) => {
         const defaultProps = {
             label: fieldName,
             name: `${FIELD_NAME}.${name}`,
         }
+        const parameterProps = {
```

Comment:
Abit unsure about this, but wanted to keep the raw `properties` from the `job-configuration`, and not conflict with label and `name` above.